docker-compose-dev.yml

```diff
@@ -5,7 +5,6 @@ services:
       context: .
       dockerfile: ./Dockerfile
       args:
-        MIX_ENV: dev
```

Comment:
For some reason with this line I have "Recv failure: Connection reset by peer" on 4000 port 🙁